webpack/assets/javascripts/react_app/components/HostDetails/Tabs/Details/Cards/Virtualization/VirtOvirt.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  DescriptionListTerm,
+  DescriptionListGroup,
+  DescriptionListDescription,
+} from '@patternfly/react-core';
+import { number_to_human_size as NumberToHumanSize } from 'number_helpers';
+import { sprintf, translate as __ } from '../../../../../../common/I18n';
+
+const VirtOvirt = ({ vm }) => (
+  <>
+    <DescriptionListGroup>
+      <DescriptionListTerm>{__('Name')}</DescriptionListTerm>
+      <DescriptionListDescription>{vm.name}</DescriptionListDescription>
+    </DescriptionListGroup>
+    <DescriptionListGroup>
+      <DescriptionListTerm>{__('VCPU(s)')}</DescriptionListTerm>
+      <DescriptionListDescription>{vm.cores}</DescriptionListDescription>
+    </DescriptionListGroup>
+    <DescriptionListGroup>
+      <DescriptionListTerm>{__('UUID')}</DescriptionListTerm>
+      <DescriptionListDescription>
+        {vm.href.split('/').pop()}
+      </DescriptionListDescription>
+    </DescriptionListGroup>
+    <DescriptionListGroup>
+      <DescriptionListTerm>{__('Memory')}</DescriptionListTerm>
+      <DescriptionListDescription>
+        {NumberToHumanSize(vm.memory, { strip_insignificant_zeros: true })}
+      </DescriptionListDescription>
+    </DescriptionListGroup>
+    <DescriptionListGroup>
+      <DescriptionListTerm>{__('Display')}</DescriptionListTerm>
+      <DescriptionListDescription>
+        {`${vm.display.type} (${vm.display.port})`}
+      </DescriptionListDescription>
+    </DescriptionListGroup>
+    {vm.display.type === 'vnc' && (
+      <DescriptionListGroup>
+        <DescriptionListTerm>{__('Keyboard')}</DescriptionListTerm>
+        <DescriptionListDescription>
+          {vm.display.keyboard_layout}
+        </DescriptionListDescription>
+      </DescriptionListGroup>
+    )}
+    {Object.values(vm.interfaces_attributes).map((nic, index) => (
+      <DescriptionListGroup key={index}>
+        <DescriptionListTerm>{__('NIC')}</DescriptionListTerm>
+        <DescriptionListDescription>
+          {`${nic.name} - ${nic.mac}`}
+        </DescriptionListDescription>
+      </DescriptionListGroup>
+    ))}
+    {Object.values(vm.volumes_attributes).map((vol, index) => (
+      <DescriptionListGroup key={index}>
+        <DescriptionListTerm>{__('Disk')}</DescriptionListTerm>
+        <DescriptionListDescription>
+          {sprintf(__('using %s'), NumberToHumanSize(vol.size))}
```

Comment:
Is this really correct? The `vol.size` suggests it's the size of the disk, not usage which would suggest:
```suggestion
          {NumberToHumanSize(vol.size)}
```